packages/rpc-graphql/src/schema/account/query.ts

```diff
@@ -0,0 +1,13 @@
+import { Base58EncodedAddress } from '@solana/addresses';
+import { Commitment } from '@solana/rpc-core';
+
+export type AccountQueryArgs = {
+    address: Base58EncodedAddress;
+    commitment?: Commitment;
+    dataSlice?: {
+        offset: number;
+        length: number;
+    };
+    encoding?: 'base58' | 'base64' | 'base64+zstd' | 'jsonParsed';
+    minContextSlot?: bigint;
+};
```

Comment:
It's annoying but not surprising that we can't just re-use `rpc-core` types for this stuff

How do you think we should deal with invalid combinations of args though? Eg encoding=base64 and dataSlice is defined. `rpc-core` shouldn't let you call `getAccountInfo` with that, but this would let you call the graphql method with it.

Reply:
As per your comment about the error handling from the RPC call, I'll add a change to bubble those throws up, so if `getAccountInfo` throws on an invalid combination, you'll get that from GraphQL. 

However, if we want the type-checking to enforce this, unfortunately we'll have to re-write that bit.

I'm exploring re-using more of the types from `rpc-core` now.

Reply:
I managed to import a few more types from `@solana/rpc-core`, but the annoying part is we have to re-create them as GraphQL objects 😡 😡